### PR TITLE
py-zipfile-deflate64: add py312 subport

### PR DESCRIPTION
**Pull request by markmentovai**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.2.1 23C71 arm64
Xcode 15.2 15C500b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Review by catap — APPROVED** [review]
Base on https://github.com/brianhelba/zipfile-deflate64/pull/41 no need for any changes

File: python/py-zipfile-deflate64/Portfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ checksums           rmd160  7d693e28bc992f6ede619771990451fb23a96d7e \
                     sha256  875a3299de102edf1c17f8cafcc528b1ca80b62dc4814b9cb56867ec59fbfd18 \
                     size    62896
 
-python.versions     38 39 310 311
+python.versions     38 39 310 311 312
 python.pep517       yes
```
Comment (reneeotten) [inline python/py-zipfile-deflate64/Portfile:22]: this is the default for subport for Python 3.7+ and thus can/should be removed

**Comment by reneeotten** [comment]
> Base on [brianhelba/zipfile-deflate64#41](https://github.com/brianhelba/zipfile-deflate64/pull/41) no need for any changes

sorry @catap - what do you mean by this comment? Do you approve the change to add the py312 subport or is there anything you're concerned about?

File: python/py-zipfile-deflate64/Portfile (hunk 1 above)
Comment (markmentovai) [reply python/py-zipfile-deflate64/Portfile]: https://github.com/macports/macports-ports/pull/22207#discussion_r1456751454:
> this is the default for subport for Python 3.7+ and thus can/should be removed

Removed.

**Comment by markmentovai** [comment]
https://github.com/macports/macports-ports/pull/22207#issuecomment-1897682095:
> > Base on [brianhelba/zipfile-deflate64#41](https://github.com/brianhelba/zipfile-deflate64/pull/41) no need for any changes
> 
> sorry @catap - what do you mean by this comment? Do you approve the change to add the py312 subport or is there anything you're concerned about?

I understood this to be another independent verification and confirmation that the existing codebase builds and works properly with Python 3.12 with no additional changes.

**Comment by catap** [comment]
> > sorry @catap - what do you mean by this comment? Do you approve the change to add the py312 subport or is there anything you're concerned about?
> 
> I understood this to be another independent verification and confirmation that the existing codebase builds and works properly with Python 3.12 with no additional changes.

I mean exactly that!

**Pull request merged**